test/unit/parser/parser_black.cpp

```diff
@@ -86,11 +86,11 @@ class TestParserBlackParser : public TestInternal
   void tryGoodInput(const std::string goodInput)
   {
     d_symman.reset(new SymbolManager(d_solver.get()));
-    Parser* parser = ParserBuilder(d_solver.get(), d_symman.get(), "test")
-                         .withStringInput(goodInput)
+    Parser* parser = ParserBuilder(d_solver.get(), d_symman.get())
```

Comment:
Shouldn't we use `std::unique_ptr` here as well? (and a few times below...)

Reply:
Done, thanks!